specs/Standards/NonFungibleToken/Payout.md

```diff
@@ -1,25 +1,30 @@
 # Standard for a Multiple-Recipient-Payout mechanic on NFT Contracts (NEP-199)
+
 Version `2.0.0`.
```

Comment:
feels like we should bump this a major version?

Reply:
This shouldn't be a breaking change.
```suggestion
Version `2.1.0`.
```